exchange/src/exchange/algos/dev_mode/dev_mode.cpp

```diff
@@ -15,8 +15,11 @@ DevModeAlgoInitializer::initialize_trader_container(
     traders::TraderContainer& traders, double start_capital
 ) const
 {
-    for (const fs::path& filepath : algo_filepaths_)
-        traders.add_trader<traders::LocalTrader>(filepath, start_capital);
+    for (const fs::path& filepath : algo_filepaths_) {
+        auto ext = filepath.extension();
+
+        traders.add_trader<traders::LocalTrader>(filepath, start_capital, ext != ".py");
```

Comment:
Not a fan of ext != ".py". We should be exact - should a .html file be a binary?

Reply:
Issue is that exes on linux don't have an extension by default...